scanner/cmd/scannerctl/scale.go

```diff
@@ -135,7 +141,8 @@ func scaleCmd(ctx context.Context) *cobra.Command {
 						// Though this method both indexes and matches, we know the indexing has already completed,
 						// and this method will just verify the index still exists. We don't account for
 						// this verification's potential failures at this time.
-						_, err = scanner.IndexAndScanImage(ctx, d, auth)
+						// TODO(ROX-23898): add flag for skipping TLS verification.
```

Comment:
Isn't that what you're doing right now?

Reply:
This is going to be done in another ticket